library/axi_ad7616/axi_ad7616_ip.tcl

```diff
@@ -10,16 +10,36 @@ source $ad_hdl_dir/library/scripts/adi_ip_xilinx.tcl
 global VIVADO_IP_LIBRARY
 
 adi_ip_create axi_ad7616
+
 adi_ip_files axi_ad7616 [list \
+    "$ad_hdl_dir/library/common/ad_edge_detect.v" \
+    "$ad_hdl_dir/library/common/ad_rst.v" \
     "$ad_hdl_dir/library/common/up_axi.v" \
-    "axi_ad7616_control.v" \
+    "$ad_hdl_dir/library/xilinx/common/ad_dcfilter.v" \
+    "$ad_hdl_dir/library/common/ad_datafmt.v" \
+    "$ad_hdl_dir/library/common/up_xfer_cntrl.v" \
+    "$ad_hdl_dir/library/common/up_xfer_status.v" \
+    "$ad_hdl_dir/library/common/up_clock_mon.v" \
+    "$ad_hdl_dir/library/common/up_delay_cntrl.v" \
+    "$ad_hdl_dir/library/common/up_adc_channel.v" \
+    "$ad_hdl_dir/library/common/up_adc_common.v" \
+    "$ad_hdl_dir/library/xilinx/common/up_xfer_cntrl_constr.xdc" \
+    "$ad_hdl_dir/library/xilinx/common/ad_rst_constr.xdc" \
+    "$ad_hdl_dir/library/xilinx/common/up_xfer_status_constr.xdc" \
+    "$ad_hdl_dir/library/xilinx/common/up_clock_mon_constr.xdc" \
     "axi_ad7616_pif.v" \
     "axi_ad7616.v" ]
 
 adi_ip_properties axi_ad7616
 
 set_property company_url {https://wiki.analog.com/resources/fpga/docs/axi_ad7616} [ipx::current_core]
```

Comment:
Replace the link with https://analogdevicesinc.github.io/hdl/library/axi_ad7616/index.html

Reply:
Done.